BrainPortal/public/javascripts/userfiles.js

```diff
@@ -33,7 +33,7 @@ function cbrain_userfile_launch_bar(tool_name, file_status_text, have_selection,
 
   $('#menu_bar').after(
     $('<div class="launch_bar">')
-      .append($('<span class="info">Select some files to launch ' + tool_name + '</span>'))
+      .append($('<span class="info">Select some files to launch <span id="tool_name">' + tool_name + '</span></span>'))
```

Comment:
Ho do you know that 'tool_name' is an ID that you can use? Is it used by some other pages in CBRAIN? How about giving it a name that is really sure to be unique for that feature?